docs/assets/templates/notebooks/template-big.dockstore.yml

```diff
@@ -63,6 +63,11 @@ notebooks:
     # A value of true will automatically display the latest tag updated as default.
     # A value of false will retain the default version that has been specified via the Dockstore UI.
     latestTagAsDefault: <Boolean>
+
+    # A boolean that will control whether DOIs are automatically issued for tags. Default: True.
```

Comment:
Suggest adding the following, similar to the `publish` and `topic` properties above

>  Workflow-wide setting that will affect ALL branches/tags; only set this as needed in a main branch.